exam/security/exam_security.xml

```diff
@@ -22,11 +22,11 @@
         </record>
 
         <record id="timetable_teacher_record_rule_exam" model="ir.rule">
-	        <field name="name">Teacher Timetable Record Rule</field>
-	        <field name="model_id" ref="model_time_table"/>
-	        <field name="global" eval="True"/>
-	        <field name="domain_force">[('user_ids','=',user.id),('timetable_type','=','exam')]</field>
-	        <field name="groups" eval="[(4, ref('school.group_school_teacher'))]"/>
+            <field name="name">Teacher Timetable Record Rule</field>
+            <field name="model_id" ref="model_time_table"/>
+            <field name="global" eval="True"/>
+            <field name="domain_force">[('user_ids','=',user.id)]</field>
```

Comment:
does the field user_ids exist?

Reply:
Yes, user_ids field exist in model time.table and we have inherited time.table in exam module.